spec/helpers/tax_return_status_helper_spec.rb

```diff
@@ -49,6 +49,23 @@
     end
   end
 
+  describe "#language_options" do
+    context "with a locale set" do
+      around do |example|
+        existing_locale = I18n.locale
+        I18n.locale = :es
+        example.run
+        I18n.locale = existing_locale
```

Comment:
So long as, if the test raises e.g. RuntimeError, this last line runs and resets the locale, this is great.

Reply:
I'd like to know if a `allow(I18n).to receive(:locale).and_return(:es)` would do the job in a `before`. If so, I prefer that. Not a blocker.